sdks/dotnet.md

```diff
@@ -0,0 +1,92 @@
+# .NET SDK
+
+- NuGet Package: https://www.nuget.org/packages/BitPesa.Sdk/
+- Git Repository: https://github.com/bitpesa/bitpesa-sdk-dotnet
+
+## Example
+
+A small example illustrating authenticating and accessing the Currency Info endpoint:
+
+### C#
+
+```csharp
+using System;
+using System.Diagnostics;
+using BitPesa.Sdk.Api;
+using BitPesa.Sdk.Client;
+using BitPesa.Sdk.Model;
+
+namespace Example
+{
+    public class InfoCurrenciesExample
+    {
+        public void main()
+        {
+            Configuration configuration = new Configuration();
+            configuration.ApiKey = "<key>";
+            configuration.ApiSecret = "<secret>";
+            configuration.BasePath = "https://api-sandbox.bitpesa.co/v1";
+
+            var apiInstance = new CurrencyInfoApi(configuration);
+
+            try {
+                // Getting a list of possible requested currencies
+                CurrencyListResponse result = apiInstance.InfoCurrencies();
+                Debug.WriteLine(result);
+            } catch (ApiException e)
+            {
+                if (e.IsValidationError) {
```

Comment:
there's not much validation error that can happen on the get currency endpoint - not sure if we want to add some examples as well where we're actually creating something - like a sender